packages/cli/__tests__/zosfiles/__system__/upload/dtu/cli.dir.upload.dtu.system.test.ts

```diff
@@ -53,37 +52,44 @@ describe("Upload directory to USS", () => {
     });
 
     afterAll(async () => {
+        // Clean up the USS directory
+        try {
+            const ussEndpoint: string = ZosFilesConstants.RESOURCE + ZosFilesConstants.RES_USS_FILES + ussname;
+            await ZosmfRestClient.deleteExpectString(REAL_SESSION, ussEndpoint, [{"X-IBM-Option": "recursive"}]);
```

Comment:
Should we use the `Delete.ussFile` API here?
Or better yet... the `deleteFiles` from the cli-test-utils 😋

Reply:
im remembering why i left this test like this.. it wasn't very easy to identitfy the name of the file and there were many tests to change. but i will make this change